### DISCUSSION TITLE
Aiortc is pushed to the SRS streaming media server, and the picture flashes and blurs

### DISCUSSION THREAD

**Discussion by robin2099** [aiortc/aiortc#625] [Q&A]
![stream](https://user-images.githubusercontent.com/46782131/140472782-0244d424-b5f0-46fd-90af-1fab25bf350d.png)
修改webcam示例，与srs建立连接，推送普通视频文件画面正常，推送摄像头收据，画面闪烁模糊

**Comment by robin2099** [comment]
![tr](https://user-images.githubusercontent.com/46782131/140473197-ddf482c8-5d43-428e-875d-151316409401.png)

**Comment by jlaine** [comment]
I'm sorry but could you provide an English description?

**Comment by robin2099** [comment]
Sorry, my English is not very good. I try to describe the problem clearly in English. Docker installs srs4 and uses webcam sample code to push it to the SRS server. The picture is very vague as shown in the figure above. You can get a clear picture by directly pushing MP4 files. What's the problem? The python script tested has been uploaded
[client.py.zip](https://github.com/aiortc/aiortc/files/7647318/client.py.zip)
